ert3/engine/_export.py

```diff
@@ -30,7 +30,15 @@ def _prepare_export(
         assert len(data) == ensemble_record.ensemble_size
         for realization, record in zip(data, ensemble_record.records):
             assert record_name not in realization[data_type]
-            realization[data_type][record_name] = record.data
+
+            if (
```

Comment:
Actually I think the blob records are not fit at all for being exported into a JSON file. I would expect them to be potentially GB's of data. I suggest we for now just skip blob records when doing this export...